docfx.json

```diff
@@ -29,10 +29,22 @@
         "dest": "csharp/language-reference/language-specification",
         "exclude": [
           "README.md",
-          "LICENSE.md",
-          "**/includes/**"
+          "LICENSE.md"
         ]
       },
+      {
+        "files": [
+          "records.md",
+          "csharp-7.0/*.md",
+          "csharp-8.0/patterns.md"
```

Comment:
This one is not on the TOC: https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/patterns?branch=pr-en-us-11026